internal/versions/versions.go

```diff
@@ -18,32 +17,50 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/thoas/go-funk"
 	"golang.org/x/sync/semaphore"
+	"gorm.io/gorm"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const releaseImageURLPrefix string = "quay.io/openshift-release-dev/ocp-release:"
```

Comment:
Not sure what this is being used for but it would be worth considering that on-prem we allow users to specify their own release image through ClusterImageSet